thoth-kafka-goodies/src/main/java/fr/maif/kafka/consumer/ResilientKafkaConsumer.java

```diff
@@ -67,6 +162,57 @@
                     .subscription(subscription)
                     .build();
         }
+        
+        public static <K,V> ConfigBuilder<K, V> builder(){
+            return new ConfigBuilder<>();
+        }
+
+        public Config<K,V> withMinBackoff(Duration minBackoff) {
```

Comment:
According to @With documentation, these methods should return copy of the object with modified value, note the same object : https://projectlombok.org/features/With

Reply:
Using the toBuilder method this should be pretty easy to implement